futures-executor/tests/local_pool.rs

```diff
@@ -404,7 +404,7 @@ fn tasks_are_scheduled_fairly() {
 }
 
 // Tests that the use of park/unpark in user-code has no
-// effect on the expected behaviour of the executor.
+// effect on the expected behavior of the executor.
```

Comment:
"behaviour" is not actually wrong (it is British English), but we already use "behavior" (American English) in most places, so prefer to use "behavior".

Reply:
Or we could use "behaviour" everywhere :wink: